content/english/ml-machine-learning/_index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Machine Learning: Linear Regression"
+description: "Understanding how to predict things."
+difficulty: "Advanced"
+prereq: "Python"
+icon: ""
+draft: false
+weight: 0
+---
+
```

Comment:
You could add a tip here recommending they go through our python workshop prior to this if they haven't. An example in another workshop: https://workshops-dev.nuevofoundation.org/java-tictactoe/